src/components/MyActivities/pages/Loggedin/MyActivitiesHelp.js

```diff
@@ -0,0 +1,42 @@
+import React, { useState } from 'react';
+import '../../MyActivitiesStyle.scss';
+import help from './help';
+
+function MyActivitiesHelp() {
+  const [helpId, setHelpId] = useState(0);
+
+  return (
+    <>
+      {localStorage.getItem('help-visited') !== 'true' ? (
+        <>
+          <div
+            className={
+              helpId == 0
+                ? 'overlay-help overlay-qr-code'
```

Comment:
Great idea to change the class according the id.
You are changing last part of the second class here. It would be better to put "qr-code" and "camera" in your conditions. I think it makes your code more readable.